bin/rails

```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path("../spring", __FILE__)
```

Comment:
It wasn't in use afaik, it was from a removed gem that was introduced accidentally.